src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/UpdateHelpWithFeeRefNumberHandlerTest.java

```diff
@@ -58,6 +60,8 @@ void shouldUpdateHwFReferenceNumberSuccessfully_FeeType_ClaimIssued() {
             CaseData data = getCaseData(response);
             Assertions.assertThat(data.getCaseDataLiP().getHelpWithFees().getHelpWithFeesReferenceNumber()).isEqualTo("7890");
             Assertions.assertThat(data.getClaimIssuedHwfDetails().getHwfReferenceNumber()).isNull();
+            Assertions.assertThat(data.getBusinessProcess().getCamundaEvent()).isEqualTo(NOTIFY_LIP_CLAIMANT_HWF_OUTCOME.toString());
```

Comment:
Assertions keyword in not required in each line as it is statically imported above. Remove from all places

Reply:
Done